### DISCUSSION TITLE
Is the public sector using this tool?

### DISCUSSION THREAD

**Discussion by bengregori** [neherlab/covid19_scenarios#472] [General]
Hey team - this tool looks like it could be incredibly powerful for public health agencies.  Is anyone aware of any public sector agencies using this? Would be great to know what type of adoption this tool has seen from government

**Comment by tleonard6** [comment]
Hello, there is a desire for the public sector to use it.  I'm working on a way to integrate it into our daily reports for some U.S. sectors.

**Reply by bengregori** [reply]
@tleonard6 Thanks for the heads up! I work for an organization that promotes the use of open source in gov't, particularly in an effort to improve reusability and create more interoperability/digital excellence in gov't operations.  We're building our own repo of projects used by gov'ts to fight covid-related challenges. I'll keep an eye out on this projects, but we'd love to include covid19 scenarios once gov't picks it up :D

https://github.com/newamericafoundation/pandemic-response-repository
